Add KDE wallpapers

yselkowitz opened the pull request:
The specfile change will also need to be propogated to src.fp.o.

yselkowitz committed:
Add KDE wallpapers

tdawson commented:
I like this, but I didn't have the -kde section in because it's not there in CentOS Stream or RHEL.
I'm ok, if others are ok.  I just didn't want others to think I was being favorable to KDE due to ... me liking KDE.

yselkowitz commented:
EPEL will need to have something like this to fix the default background situation there, but while RHEL and EPEL are separate, ELN+Extras has one source, so I don't see a reason not to do it here.

However, we could wait and see what the EPEL solution looks like, if it won't be forever until that happens (I might just look into it myself).

yselkowitz commented:
I have a prototype for the EPEL wallpaper problem, which may effect this slightly, so let's hold off on this a bit.

yselkowitz commented:
Moved to draft and modified to be compatible with my EPEL solution currently available for testing at https://copr.fedorainfracloud.org/coprs/yselkowitz/epel-backgrounds-kde/

yselkowitz commented:
Per discussions today, we're going to take another approach.
